confgenerator/files.go

```diff
@@ -59,7 +59,7 @@ func (uc *UnifiedConfig) GenerateFilesFromConfig(ctx context.Context, service, l
 			}
 		}
 	case "otel":
-		otelConfig, err := uc.GenerateOtelConfig(ctx)
+		otelConfig, err := uc.GenerateOtelConfig(ctx, outDir)
```

Comment:
[nit] In the fluent bit case, the `Generate` Command just returns the files and then we write in this function. For OTel though, it looks like we write the file in side this call. Lets make the behaviour consistent between the two?

Reply:
Can do on a follow up

Reply:
No, we don't write the otel generate configs inside `GenerateOtelConfig`, they are written in here the same way as the `fluent-bit` case (see the next 10 lines).

In the PR, @rafaelwestphal is passing `outDir` to know where did the `*_oltp.json` files were written too.